### DISCUSSION TITLE
AWS - EC2, RDS 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#18] [Q&A] [answered]
AWS - EC2, RDS 란?

**Comment by tjdud22** [comment, marked as answer]
RDS(Relational Database Service)란 무엇인가?
간단히 말하면 관계형 데이터베이스를 제공하는 AWS의 서비스이다. 유저가 사용하기 쉽도록 인프라 등을 자동화 시켜주고 유저들은 앤드포인트로 접속할 수 있는 데이터베이스를 제공받는다.

EC2는 AWS에서 제공하는 클라우드 컴퓨팅 서비스다.
